app/assets/javascripts/active_admin.js

```diff
@@ -0,0 +1,2 @@
+//= require active_admin/base
+//= require activeadmin_addons/all
```

Comment:
Expected space or tab after '//' in comment  spaced-comment

Reply:
Estas seguro que va a así? o el mono esta molestando de puro pesado?

Reply:
si, el mono alega con las notaciones de rails en js

Reply:
Perfect, entonces aproba3